crates/iceberg-catalog/migrations/20250207162731_endpoint_stats.sql

```diff
@@ -0,0 +1,89 @@
+create type api_endpoints as enum (
+    'catalog-post-aws-s3-sign',
+    'catalog-post-prefix-aws-s3-sign',
+    'catalog-get-config',
+    'catalog-get-namespaces',
+    'catalog-post-namespaces',
+    'catalog-get-namespace',
+    'catalog-head-namespace',
+    'catalog-post-namespace',
+    'catalog-delete-namespace',
+    'catalog-post-namespace-properties',
+    'catalog-get-namespace-tables',
+    'catalog-post-namespace-tables',
+    'catalog-get-namespace-table',
+    'catalog-post-namespace-table',
+    'catalog-delete-namespace-table',
+    'catalog-head-namespace-table',
+    'catalog-get-namespace-table-credentials',
+    'catalog-post-tables-rename',
+    'catalog-post-namespace-register',
+    'catalog-post-namespace-table-metrics',
+    'catalog-post-transactions-commit',
+    'catalog-post-namespace-views',
+    'catalog-get-namespace-views',
+    'catalog-get-namespace-view',
+    'catalog-post-namespace-view',
+    'catalog-delete-namespace-view',
+    'catalog-head-namespace-view',
+    'catalog-post-views-rename',
+    'management-get-info',
+    'management-post-bootstrap',
+    'management-post-role',
+    'management-get-role',
+    'management-post-role-id',
+    'management-get-role-id',
+    'management-delete-role-id',
+    'management-post-search-role',
+    'management-get-whoami',
+    'management-post-search-user',
+    'management-post-user-id',
+    'management-get-user-id',
+    'management-delete-user-id',
+    'management-post-user',
+    'management-get-user',
+    'management-post-project',
+    'management-get-default-project',
+    'management-delete-default-project',
+    'management-post-rename-project',
+    'management-get-project-id',
+    'management-get-endpoint-statistics',
+    'management-delete-project-id',
+    'management-post-warehouse',
+    'management-get-warehouse',
+    'management-get-project-list',
+    'management-get-warehouse-id',
+    'management-delete-warehouse-id',
+    'management-post-warehouse-rename',
+    'management-post-warehouse-deactivate',
+    'management-post-warehouse-activate',
+    'management-post-warehouse-storage',
+    'management-post-warehouse-storage-credential',
+    'management-get-warehouse-statistics',
+    'management-get-warehouse-deleted-tabulars',
+    'management-post-warehouse-deleted-tabulars-undrop1',
+    'management-post-warehouse-deleted-tabulars-undrop2',
+    'management-post-warehouse-delete-profile',
+    'management-get-permissions',
+    'management-post-permissions',
+    'management-head-permissions',
+    'management-delete-permissions');
+
+
+create table endpoint_statistics
```

Comment:
Just a comment that we don't for get to add the primary access as index.

Reply:
added 3 indexes